0000-stream/0000-stream.md

```diff
@@ -0,0 +1,182 @@
+---
+title: STREAM: A Multiplexed Money and Data Transport Protocol
+draft: 1
+---
+
+# STREAM: A Multiplexed Money and Data Transport Protocol
+
+ILP/STREAM, or the STREAMing Transport for the Real-time Exchange of Assets and Messages, is an Interledger transport protocol that provides for multiplexed streams of money and data over a virtual ILP connection. STREAM is a successor to [PSK2](../0025-pre-shared-key-2/0025-pre-shared-key-2.md) and takes significant inspiration from the [QUIC](https://tools.ietf.org/html/draft-ietf-quic-transport-10) Internet transport protocol.
+
+## Definitions
+
+- **Client** - One of the two entities using STREAM. The Client initiates a Connection to the Server using STREAM packets sent over ILP. Because STREAM uses bidirectional streams, the Client may be either the Sender or the Receiver of a given packet.
```

Comment:
Do the `two entities` include `Server`?
I could not make it clear that...
- Two entities mean `sender/receiver` AND `server`
- Two entities mean `sender` AND `receiver`, and do not contain `server`

Reply:
Also, I could not understand what `initiates` means here because `Setup` section explains the `initiation`, doesn't it?

In the section, it seems that we don't use ILP packet for the `initiation`.